torch/testing/_internal/common_utils.py

```diff
@@ -974,20 +974,20 @@ def setUp(self):
 
         set_rng_seed(SEED)
 
-    def genSparseTensor(self, size, sparse_dim, nnz, is_uncoalesced, device='cpu'):
+    def genSparseTensor(self, size, sparse_dim, nnz, is_uncoalesced, device='cpu', dtype=torch.double):
```

Comment:
I'd suggest requiring device and dtype, like make_tensor(), but if that's a pain for this PR then can we make the default 'cpu' and 'torch.float'?

Reply:
Sure, I have a question, why the default `dtype` value should be `torch.float` if the default dtype for test_sparse and test_autograd  is `torch.double`
https://github.com/pytorch/pytorch/blob/0e03a2978ad5ed2be09f3e4691b2d04d6d4c6c90/test/test_sparse.py#L5